gui/slick/js/ajaxEpSearch.js

```diff
@@ -1,4 +1,4 @@
-var searchStatusUrl = srRoot + '/home/getManualSearchStatus';
+var searchStatusUrl = 'home/getManualSearchStatus';
```

Comment:
I think this PR needs to wait for the BASE_URL one to be merged as that'll fix this.

@p0psicles